packages/@uppy/react/src/useUppy.js

```diff
@@ -17,7 +17,7 @@ module.exports = function useUppy (factory) {
 
   useEffect(() => {
     return () => {
-      uppy.current.close()
+      uppy.current.close({ reason: null })
```

Comment:
Explicitly passing `null` still feels a bit off to me. You would expect a nullish value to be the default.

Reply:
True. Which reason should we use? `cleanup` ?

Reply:
Cleanup also sounds inherit to `close` in itself. In the end it is the difference of human/user vs. machine. So perhaps `reason: 'unmount'` does make sense after all.

Reply:
Wouldn't we then bring a concept of react into core, which would break the separation of concerns?

Reply:
Has nothing to do with React per se, you can unmount in vanilla JS (remove it programmatically), vue, svelte, angular, whatever.